docs/loadflow/validation.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Load-flow validation
+layout: default
+---
+
+The load-flow validation aims at ensuring the consistency of load-flow results with respect to a set of rules that
+describes what is an *acceptable* load-flow result. On the most abstract level, a load-flow result is *acceptable* if it
+describes a feasible steady-state of a power system given its physics and its logics. More practically, generations of
+practitioners have set quasi-standard ways to describe them that allows to define precise rules.
+
+Overall, tests should never be too tight and leniency is preferred to tightness in case approximations are needed or in
+case expectations are unclear (typically when the input data is inconsistent). For example, there is a switch to test
+only the main component because it is not clear what to expect from load flow results on small connected components.
+
+Another important global setting is the `ok-missing-values` parameter, which determines which is OK to have missing
+values or `NaN`. Normally, it should be set to false but it may be useful in cases the power flow results are incomplete.
```

Comment:
*Another important global setting is the `ok-missing-values` parameter, which determines **if** is OK to have missing values or `NaN`. Normally, it should be set to false but it may be useful **in the cases where** the power flow results are incomplete.